composer/trainer/trainer_hparams.py

```diff
@@ -792,7 +791,7 @@ def initialize_object(self, model: ComposerModel, dataloader_hparams: DataLoader
 
         # Metrics
 
-        # TODO(Ravi): Cleanup this code as part of the MetricsModule. This code was copied
+        # TODO(Ishana): Cleanup this code as part of the MetricsModule. This code was copied
```

Comment:
Is there a Jira issue to track this TODO?  If not can we file an issue and remove the TODO from code?